submissions/drewzag/memory-pair-game/app.js

```diff
@@ -0,0 +1,87 @@
+const section = document.querySelector('section')
+
+const getData = () => [
+  { imgSrc: './images/ari.webp', id: 'arisona' },
+  { imgSrc: './images/cle.webp', id: 'clevlend' },
+  { imgSrc: './images/lv.webp', id: 'las-vegas' },
+  { imgSrc: './images/mia.webp', id: 'miami' },
+  { imgSrc: './images/sea.webp', id: 'seattle' },
+  { imgSrc: './images/tb.webp', id: 'tampa-bay' },
+]
+
+const randomize = () => {
+  const cardData = getData().concat(getData())
+  cardData.sort(() => Math.random() - 0.5)
+  return cardData
+}
+
+const cardGenerator = () => {
```

Comment:
the function always means some action, so the name should start from a verb, e.g. `generateCards`